src/overmap.cpp

```diff
@@ -3713,13 +3660,14 @@ const unsigned &oter_id::operator=(const int &i)
     return _val;
 }
 // ter(...) = "rock"
-oter_id::operator std::string() const
+namespace { std::string const debug_dummy_string; }
```

Comment:
You've got an interesting point there, I wasn't aware of that.

But the static variable could be directly after the debug message, which requires user input and is therefor not performance relevant code. And it should not happen anyway, so the static variable (inside the if block) should never be initialized.